shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md

```diff
@@ -0,0 +1,38 @@
+# Optimization of a Solid Hook
+Optimization of a solid 3D Hook subjected to multiple constraints. 
+
+**Author**: Armin Geiser  
+
+**Kratos version**: 9.0
+
+## Optimization Problem
+- Objective
+    - Minimize mass
+
+- Constraints
+    - constraint 1  &nbsp; : &nbsp; strain energy of main &le; initial value for load case 1
```

Comment:
Please don't use these spacings (`&nbsp`) makes the raw text hard to read and is usually not needed.

same holds for items below.

Reply:
```suggestion
    1. strain energy of main load (LC1) &le; initial value
```